backend/common/Cargo.toml

```diff
@@ -1,29 +1,21 @@
 [package]
-name = "telemetry"
-version = "0.3.0"
+name = "common"
+version = "0.1.0"
 authors = ["Parity Technologies Ltd. <admin@parity.io>"]
 edition = "2018"
-license = "GPL-3.0"
```

Comment:
no license?

Reply:
Ah whoops; I regenerated the file so must have forgot to put that back; thanks for spotting!